src/index.css

```
@@ -1,3 +1,7 @@
+:root {
+  --Azul:#198ffb;
```

¡Muy bien con el uso de variables de CSS!
```suggestion
  --Azul: #198ffb;
```